src/platform/forms-system/test/js/components/EditContactInfo.unit.spec.jsx

```diff
@@ -39,43 +41,59 @@ describe('EditContactInfo', () => {
       );
 
       expect(getByText(content.editHomePhone)).to.exist;
-      const phoneNumber = getByLabelText('Home phone number', { exact: false });
+      const phoneNumber = $(
+        'va-text-input[label^="Home phone number"]',
+        container,
+      );
       expect(phoneNumber).to.exist;
-      expect(container.innerHTML).to.include('*Required');
-      expect(getByLabelText('Extension', { exact: false })).to.exist;
+      expect(phoneNumber.required).to.be.true;
+      const extension = $('va-text-input[label^="Extension"]', container);
+      expect(extension).to.exist;
       expect(getByText('Save')).to.exist;
```

Comment:
why is this not updated like the cancel button?

Reply:
The "Save" button uses the React `LoadingButton` component - it includes a loading indicator inside after you click on it. Sadly, the component library doesn't include a [`va-button-icon` with a loading icon](https://design.va.gov/storybook/?path=/docs/components-va-button-icon--change-file), but Adam has opened a ticket to get it added. So once it's added all of the "Save" button selectors will need to be updated. So, I didn't want to touch them for now.